src/services/utility-helpers.js

```diff
@@ -556,10 +556,11 @@ export function autoScroll(currentItem, containerRef, toTop = false) {
  * @param {Object} event keydown event
  * @param {String} id player instance ID in VideoJS
  * @param {Boolean} canvasIsEmpty flag to indicate empty Canvas
- * @returns 
+ * @returns {Boolean} player's paused status for global state update
```

Comment:
Yes, I think it makes sense to use the suggested pattern or even a enum as the output of this function. As you mentioned this has more potential outcomes than a boolean value can handle.
In the current implementation the return value only evaluates to either `true` or `false`. I think a enum would probably be a suitable data type to handle this.

Reply:
I changed the return value to a enum, so that it's easy to handle on the other side using a simple switch case as needed for each result caused by the hotkey actions.